docs/.gitignore

```diff
@@ -0,0 +1,191 @@
+# Covers Maven specific
```

Comment:
I.e. Maven in the gitignore seems a bit odd or has this any purpose?

Reply:
its just our default .gitignore, that we have in every repository. We have #36 to combine them but we are still waiting for the webcomponent templates.